src/protocol/attribution/accumulate.rs

```diff
@@ -0,0 +1,322 @@
+use super::{AccumulationInputRow, AccumulationOutputRow, AttributionInputRow};
+use crate::{
+    error::BoxError,
+    field::Field,
+    helpers::{fabric::Network, prss::SpaceIndex},
+    protocol::{
+        batch::{Batch, RecordIndex},
+        context::ProtocolContext,
+        RecordId, Step,
+    },
+    secret_sharing::Replicated,
+};
+use futures::future::{try_join, try_join_all};
+
+/// Accumulation step for Oblivious Attribution protocol.
+#[allow(dead_code)]
+pub struct Accumulate<'a, F> {
```

Comment:
Renamed from `Attribution` to `Accumulate` to better reflect what it is actually doing. There are three steps in the oblivious attribution. 1. accumulation (this step), 2. sorting, 3. aggregation. I will create Attribution trait in future PR when I implement the rest of the steps.

Reply:
Maybe AccumulateCredit? Since "accumulate" is kind of a general term.